src/core/util.ts

```diff
@@ -24,7 +24,28 @@ export const HexStr = {
   isHex(value: any, length?: number): boolean {
     return bytes.isHexString(value, length);
   },
+  isSameHex( a:string, b:string ): boolean {
```

Comment:
If it's for the address comparison, ethers.utils.getAddress() and simple `==` would suffice.

Reply:
Good. I will update in web3klaytn.